nodejs/packages/layer/src/wrapper.ts

```diff
@@ -69,6 +70,7 @@ const instrumentations = [
   new NetInstrumentation(),
   new PgInstrumentation(),
   new RedisInstrumentation(),
```

Comment:
Should these all be moved into a default implementation of `configureInstrumentations()` such that users have a way to ask for less auto-instrumentation and not just more?  Minimizing the impact of auto-instrumentation on cold start times is a recurring theme we're hearing from customers.

Reply:
Suits me. Done.

Note, I have no good way to test this.